protocol/wire.go

```diff
@@ -80,13 +80,11 @@ func (e *InvalidTrace) Error() string {
 // ValidTrace takes in an SSF span and determines if it is valid or not.
 // It also makes sure the Tags is non-nil, since we use it later.
```

Comment:
Wow, that doc comment (and the one on `ValidateTrace` below) is kinda a total lie (has been since it was introduced, I believe: https://github.com/stripe/veneur/blame/2624d78c6b04aeb1dba1b0e6302c94d3062d4d9c/samplers/parser.go#L56

Maybe update to say something like 

> ValidTrace returns true if an SSF span contains all the data necessary to interpret the span as part of a trace.

?